forgerock-auth-ui/src/main/res/layout/fragment_device_binding_callback.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2019 - 2022 ForgeRock. All rights reserved.
+  ~
+  ~ This software may be modified and distributed under the terms
+  ~ of the MIT license. See the LICENSE file for details.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".callback.DeviceProfileCallbackFragment" android:id="@+id/frameLayout">
```

Comment:
Change the context to DeviceBindingCallbackFragment

Reply:
Oops... Wow, eagle eyes you have! 🥇 Fixed that...